script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -278,7 +278,7 @@ It's after seven in the evening.
 [sha:c80a8c954dcd3ec80aba3cdf3d84d0280e2c2229]{
 -- Page 923, Offset 8112.
 -- 屋敷の門限は８時だから、急いで帰らないとまた秋葉に小言を言われる。
-Curfew at the mansion is eight o'clock, so I'd better get a move on if I don't want Akiha to give me a hard time again.
+Curfew at the mansion is eight o'clock, so I'd better get a move on if I don't want another round of chiding from Akiha.
```

Comment:
Chiding kinda mild in general, would be cool if we could use the full breadth of the English language here to make him sound more realistic